city_scrapers/spiders/chi_ssa_48.py

```diff
@@ -0,0 +1,92 @@
+from datetime import datetime
+
+from city_scrapers_core.constants import COMMISSION
+from city_scrapers_core.items import Meeting
+from city_scrapers_core.spiders import CityScrapersSpider
+
+
+class ChiSsa48Spider(CityScrapersSpider):
+    name = "chi_ssa_48"
+    agency = "Chicago Special Service Area #48 Old Town"
+    timezone = "America/Chicago"
+    start_urls = ["https://oldtownchicago.org/ssa-48/"]
+
+    def parse(self, response):
+        """
+        `parse` should always `yield` Meeting items.
+
+        Change the `_parse_title`, `_parse_start`, etc methods to fit your scraping
+        needs.
+        """
+        meeting_year = response.xpath("//div[@class='meeting-dates-block']/h2")
+        meeting_date = response.xpath("//div[@class='meeting-dates-block']/h5")
+        meeting_info = response.xpath("//div[@class='meeting-dates-block']/p")
+
+        for item_date, item_info in zip(meeting_date, meeting_info):
+            meeting = Meeting(
+                title="Special Service Area 48 Meeting",
+                description="",
+                classification=COMMISSION,
+                start=self._parse_start(item_date, item_info, meeting_year),
+                end=self._parse_end(item_date, item_info, meeting_year),
+                all_day=False,
+                time_notes="",
+                location=self._parse_location(item_info),
+                links=[],
+                source=self._parse_source(response),
+            )
+
+            meeting["status"] = self._get_status(meeting, meeting_info.xpath(".//text()").get())
+            meeting["id"] = self._get_id(meeting)
+
+            yield meeting
+
+    def _parse_start(self, date, info, year):
```

Comment:
Since most of the functionality is shared with `_parse_end`, it might make sense to have a single `_parse_start_end` function that returns a tuple of `(start, end)`

Reply:
Thank you for your code review. In order to easily match meeting minutes with particular meeting, I added also a return of parsed meeting date to this function. Other comments should be also resolved.